nodeList.json

```diff
@@ -0,0 +1,900 @@
+[
```

Comment:
The code i got from Pedro seems to reference it as an array, so the site is coded to look for an array. I would have to go back and change the code to treat it as an object.